tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -873,20 +873,21 @@ $ sudo pip install /tmp/tensorflow_pkg/tensorflow-0.12.1-py2-none-any.whl
 ## Optimizing CPU performance
 
 To be compatible with as wide a range of machines as possible, TensorFlow
-defaults to only using SSE4.1 SIMD instructions on x86 machines. Most modern PCs
+defaults to only using SSE4 SIMD instructions on x86 machines. Most modern PCs
```

Comment:
when I think about it, we only build with -c opt for now.
What does that bring by default?